campaignion_wizard/src/WebformStepUnique.php

```diff
@@ -20,7 +20,7 @@ public function validateStep($form, &$form_state) {
         $c = reset($cs);
       }
     }
-    if ($c) {
+    if ($c && !$this->hasCaptcha($form['#node'])) {
```

Comment:
I don’t quite get the logic behind "if there is a capture, the email address does not have to be unique", aren’t those completely different things? For spamming a form, one could easily generate unique email addresses. On the other hand a capture won’t stop real people from submitting the form twice with the same data.

Reply:
True. But we require only one of them to be set up.